scss/dependencies/_mixins.scss

```diff
@@ -78,3 +78,23 @@
   @keyframes #{$name} { @content; }
 }
 
+/**
+* 1 - Define class name, eg `steps--count3`;
+* 2 - Except first child, all steps must be margin-left relative to wrapper and
+      quantity of node steps;
+* 3 - Define line after each step, except to the last child.
+*/
+
+@mixin steps($stepWidth, $count) {
+  $linesCount: $count - 1;
+  .#{'steps--count' + $count} { /* 1 */
+    .step {
+      margin-left: calc(((100% - #{$stepWidth * $count}) / #{$linesCount})); /* 2 */
+      &:first-child { margin-left: 0; }
+      &:last-child:after { width: 0; }
+      &:after {
+        width: calc(((100% - #{$stepWidth * $count}) / #{$linesCount}) + (#{$grid * 3})); /* 3 */
+      }
```

Comment:
`&:first-child` and `&:last-child:after` should be declared in common `.step` definition